README.md

```diff
@@ -1,10 +1,12 @@
 BeVexed 1.1
```

Comment:
While you're touching the readme, use markdown for formatting. There are markdown editors online.
"BeVexed" would be a big header.
Drop the "1.1". Future releases may not need a change in the readme, so having the version here is inconvenient.